### PR TITLE
Update the deploy guide to describe 2x week deploys

### PR DESCRIPTION
**Pull request by jmhooper**
We talked about this at our staff eng weekly and at engineering huddle. This commit updates the documentation to describe that new process

### CODE REVIEW

**Review by zachmargolis — APPROVED** [review]
LGTM

**Review by jeffthink — APPROVED** [review]
:shipit:

File: _articles/appdev-deploy.md [hunk 1]
```diff
@@ -63,7 +63,6 @@ This guide assumes that:
 Note: it is a good idea to make sure you have the latest pulled down from identity-devops - lots of goood improvements all the time!
 
 ### Pre-deploy
-Scheduled for every **Tuesday**
```
Comment (aduth) [inline _articles/appdev-deploy.md:66]: One process item that hadn't occurred to me 'til now is how can we ensure timely reviews of the deploy branch if we'll now be cutting it right before the actual deployment? Should we wait and see if that'll be a problem?

**Review by aduth — APPROVED** [review]

File: _articles/appdev-deploy.md (hunk 1 above)
Comment (mitchellhenke) [reply _articles/appdev-deploy.md:66]: I think the plan was to use the SHA on staging to deploy something that had been deployed before.

Comment (aduth) [reply _articles/appdev-deploy.md:66]: > I think the plan was to use the SHA on staging to deploy something that had been deployed before.

I'm not sure I follow. Is the pull request guidance in this article still valid with the updated workflow?

**Comment by aduth** [comment]
Superseded by #313 . (I forgot that this pull request existed, but co-authored a commit to combine content)

**Pull request closed without merging**